src/movie-area.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Card from './movie-card';
+import './movie-area.css';
+
+const Area = ({ movies }) => {
+
+  const movieCards = movies.map(movie => {
+    return (
+      <Card
+      title= {movie.title}
+      img= {movie.poster_path}
+      />
+    )
+  })
+
+  return (
+    <div className= 'area'>
+      {movieCards}
+    </div>
+  )
+}
+
+export default Area;
```

Comment:
Again, clean and simple. Maybe we could add more attributes to our JSX html tags for accessibility in the future.